.github/workflows/terraform.yml

```diff
@@ -0,0 +1,67 @@
+name: 'Terraform'
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
This pipeline only deploys to QA. We would need to create more temporary environments (review apps) to be able to enable this for PRs. We definitely want to get there, but let's leave it for now.